javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -22,15 +24,19 @@
  * @author Ruslan Sennov
  * @since 2.0.0
  */
-public final class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T>, Serializable {
-
+@SuppressWarnings({"SuspiciousArrayCast", "unchecked"})
+public class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T> {
```

Comment:
It was put back in a later commit, I just realized after pushing that I forgot to rebase the `final` back here :)
Force pushed a new version :)